### DISCUSSION TITLE
2023-11-09 Users Community Call

### DISCUSSION THREAD

**Discussion by gonuke** [svalinn/DAGMC#918] [Community Calls]
Attending:
* UW-Madison: @gonuke, @bohmt 
* Copenhagen Atomics: @ebknudsen 
* MIST: @ahnaf-tahmid-chowdhury
* Naarea:  @nplinden
* First Light Fusion: @shimwell 
* UKAEA: @makeclean 
* ANL: @pshriwise

**Comment by gonuke** [comment]
Cubit 2023.11 includes DAGMC as first class component. 

Exports surface mesh generated by Cubit. More user control of mesh generation parameters using some standard mesh generation parameters, but lacking a faceting tolerance.

Metadata assignment is now more consistent with other mesh-generation metadata operations. 

Builtin mesh export instead of using plugin. `export cf_dagmc ... `

@pshriwise will give webinar on Nov 10.

**Comment by gonuke** [comment]
@nplinden:

Challenges with imprint/merge with complex CAD-originated model.

Some discussion of best practices including experience from @bohmt

**Comment by gonuke** [comment]
@pshriwise 

Python interface to interrogate DAGMC model: https://github.com/svalinn/pydagmc

Also some work on wheels for (py)MOAB by @akoen

https://github.com/Thea-Energy/moab-python/releases/tag/0.0.01-beta1

**Comment by gonuke** [comment]
Discussion of complex model comparison/benchmarks:

@shimwell - Difficult to do a direct comparison to complex CAD models because it is hard to make a CSG equivalent for benchmarking.

@makeclean - there are some examples of complex CAD models that have been simplified for conversion to CSG - those models might be good candidates for such benchmarking.  However, those models often have segmenting to allow conversion that can have performance implications.

@gonuke - we could unit the segmented CAD models to have: 1) segmented CAD, 2) segmented CSG, and 3) untied CAD to compare across all

**Comment by shimwell** [comment]
Link to the Cubit Coreform talk tomorrow by Patrick https://us02web.zoom.us/webinar/register/WN_49YsNqLKSsibQVmYNGF3xQ#/registration
